.github/workflows/pr-receive.yml

```diff
@@ -10,7 +10,12 @@ permissions:
 jobs:
   pr-target:
     runs-on: ubuntu-latest
-    if: github.repository == 'llvm/llvm-project'
+    # Ignore PRs with more than 10 commits.  Pull requests with a lot of
+    # commits tend to be accidents usually when someone made a mistake while trying
+    # to rebase.  We want to ignore these pull requests to avoid excessive
+    # notifications.
+    if: github.repository == 'llvm/llvm-project' &&
+        github.event.pull_request.commits < 10
```

Comment:
Yes, we could add that.

Reply:
I think I'm going to add this in a separate pull request, because I want to write a separate commit message about ignoring drafts and I can't push more than two commits at once here.

Reply:
Actually, I think in order to add this, we would need to run this action when the Draft flag is removed from the PR, but I don't see an event for that.

Reply:
It took me a while to find it https://docs.github.com/en/rest/overview/issue-event-types?apiVersion=2022-11-28#ready_for_review

Reply:
PR for ignoring drafts: https://github.com/llvm/llvm-project/pull/66578